tests/src/system/basic/WskActionTests.scala

```diff
@@ -134,6 +134,34 @@ class WskActionTests
             }
     }
 
+    it should "copy an action and ensure exec, parameters, and annotations copied" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val origActionName = "orignAction"
+            val copiedActionName = "copiedAction"
+            val params = Map("a" -> "A".toJson)
+            val annots = Map("b" -> "B".toJson)
+
+            assetHelper.withCleaner(wsk.action, origActionName) {
+                val file = Some(TestUtils.getTestActionFilename("wc.js"))
+                (action, _) => action.create(origActionName, file, parameters = params, annotations = annots)
+            }
+
+            assetHelper.withCleaner(wsk.action, copiedActionName) {
+                (action, _) => action.create(copiedActionName, Some(origActionName), Some("copy"))
+            }
+
+            val originActionAnnots = wsk.action.get(origActionName, fieldFilter = Some("annotations")).stdout
+            val copiedActionAnnots = wsk.action.get(copiedActionName, fieldFilter = Some("annotations")).stdout
+            val originActionParams = wsk.action.get(origActionName, fieldFilter = Some("parameters")).stdout
+            val copiedActionParams = wsk.action.get(copiedActionName, fieldFilter = Some("parameters")).stdout
+            val originActionExec = wsk.action.get(origActionName, fieldFilter = Some("exec")).stdout
+            val copiedActionExec = wsk.action.get(copiedActionName, fieldFilter = Some("exec")).stdout
+
+            copiedActionAnnots.substring(copiedActionAnnots.indexOf("\n")) shouldBe originActionAnnots.substring(originActionAnnots.indexOf("\n"))
+            copiedActionParams.substring(copiedActionParams.indexOf("\n")) shouldBe originActionParams.substring(originActionParams.indexOf("\n"))
```

Comment:
I updated the test to make it more readable, and added some methods that will make writing tests that require examination of non-activation record responses from the CLI easier in the future.